crowdsec-docs/src/components/RemediationSupportBadges.tsx

```diff
@@ -37,15 +37,21 @@ const RemediationSupportBadge = ({ title, description, supported }: { title: str
 }
 
 export default function RemediationSupportBadges({ MTLS, Metrics, Prometheus, Mode, Appsec }: RemediationSupportBadgesProps): React.JSX.Element {
+    const mtlsSupport = MTLS ? 'Supported' : 'Unsupported';
+    const metricsSupport = Metrics ? 'Supported' : 'Unsupported';
+    const prometheusSupport = Prometheus ? 'Supported' : 'Unsupported';
+    const modeSupport = Mode ? 'Live & Stream' : 'Live only';
+    const appsecSupport = (Appsec !== undefined && Appsec) ? 'Supported' : 'Unsupported';
+
     return (
         <div className='tw-flex tw-justify-center tw-flex-wrap tw-mb-4 tw-gap-2'>
             {Appsec !== undefined && (
-                <RemediationSupportBadge title='AppSec' description='Can forward HTTP requests to the AppSec Component' supported={Appsec} />
+                <RemediationSupportBadge title='AppSec' description='Can forward HTTP requests to the AppSec Component' support={appsecSupport} />
             )}
-            <RemediationSupportBadge title='Mode' description='Can be configured in different modes, typically live or stream' supported={Mode} />
-            <RemediationSupportBadge title='Metrics' description='Can send detailed metrics to LAPI' supported={Metrics} />
-            <RemediationSupportBadge title='MTLS' description='Can do mutual TLS authentication to LAPI' supported={MTLS} />
-            <RemediationSupportBadge title='Prometheus' description='Can expose metrics to Prometheus' supported={Prometheus} />
+            <RemediationSupportBadge title='Mode' description='Can be configured in different modes, typically live or stream' support={modeSupport} />
```

Comment:
the description doesnt really match the present values anymore.

Reply:
true, I was just testing to see what looked like. but we need to do a slight improve on the Mode bubble